src/components/our_projects.rs

```diff
@@ -0,0 +1,167 @@
+use yew::{html, Component, ComponentLink, Html, ShouldRender};
+use yew::services::fetch::{FetchService, FetchTask, Request, Response};
+use super::{project_card::{ProjectCard}};
```

Comment:
The inner curly braces are not required.

```suggestion
use super::project_card::ProjectCard};
```